docs/content/guide/migration.ngdoc

```diff
@@ -15,6 +15,11 @@ which drives many of these changes.
 * Several new features, especially animations, would not be possible without a few changes.
 * Finally, some outstanding bugs were best fixed by changing an existing API.
 
+## Migrating from 1.7 to 1.8
+
+AngularJS 1.8 is a breaking change release from 1.7 to mitigate a security issue.
```

Comment:
We will need more info here. Just pointing it out so that we don't forget.

Reply:
Indeed! I didn't want to preempt your PR :-)